tests/base-notebook/test_packages.py

```diff
@@ -71,6 +71,7 @@
     "hdf5",
     "jupyterlab-git",
     "openssl",
+    "pandas[version='>1.5.3,<2.0.0",
```

Comment:
As far as I remember, it should be exactly as I wrote it in a comment above.

Reply:
"pandas[version='>" strange but I will give it a try :)

Reply:
Yep, I think the version parsing was written quite a while ago and some format has changed, so it would be nice to update that script, but that doesn't give much trouble.